specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+          "readOnly": true,
+          "description": "Earliest time to receive a signal (ISO 8601 UTC standard)."
+        },
+        "rxEndTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Time to lost receiving a signal (ISO 8601 UTC standard)."
+        },
+        "startAzimuthDegrees": {
+          "type": "number",
+          "readOnly": true,
+          "description": "Azimuth of the antenna at the start of the contact in decimal degrees."
+        },
+        "endAzimuthDegrees": {
+          "type": "number",
+          "readOnly": true,
+          "description": "Azimuth of the antenna at the end of the contact in decimal degrees."
+        },
+        "startElevationDegrees": {
+          "type": "number",
+          "readOnly": true,
+          "description": "Spacecraft elevation above the horizon at contact start."
+        },
+        "endElevationDegrees": {
+          "type": "number",
+          "readOnly": true,
+          "description": "Spacecraft elevation above the horizon at contact end."
+        }
+      },
+      "description": "Contact Instance Properties."
+    },
+    "Contact": {
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContactsProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Customer creates a contact resource for a spacecraft resource."
+    },
+    "CloudErrorBody": {
+      "type": "object",
+      "x-ms-external": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "x-ms-identifiers": [
+            "code"
+          ],
+          "description": "A list of additional details about the error."
+        }
+      },
+      "description": "An error response from the service."
+    },
+    "CloudError": {
+      "type": "object",
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "description": "An error response from the service."
+    },
```

Comment:
If its not a breaking change, I'd suggest removing these definitions and instead referring to common types:

See e.g. ErrorResponse
[https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#L300](https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#L300)
And ErrorDetail
[https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#LL258C21-L258C21](https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#LL258C21-L258C21)